02-microarray/clustering_microarray_01_heatmap.Rmd

```diff
@@ -226,8 +226,8 @@ Now we are going to use a combination of functions from base R and the `pheatmap
 
 ## Choose genes of interest
 
-Although you may want to create a heatmap including all of the genes in the set, alternatively, the heatmap could be created using only genes of interest.
-For this example, we will sort genes by variance and select genes in the upper quartile, but there are many alternative criteria by which you may want to sort your genes <i>e.g.</i> fold change, t-statistic, membership in a particular gene ontology, and so on.
+Although you may want to create a heatmap including all of the genes in the dataset, this can produce a very large image that is hard to interpret. Alternatively, the heatmap could be created using only genes of interest.
```

Comment:
```suggestion
Although you may want to create a heatmap including all of the genes in the dataset, this can produce a very large image that is hard to interpret. 
Alternatively, the heatmap could be created using only genes of interest.
```